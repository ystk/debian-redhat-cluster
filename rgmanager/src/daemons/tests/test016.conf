<?xml version="1.0"?>
<!--
     Negative test: Attempting to assign an IP address to multiple resource
     groups.  Only ONE copy of "192.168.1.3" should appear in the tree.

     There should be _no_ NEEDSTOPs/NEEDSTARTs anywhere.
-->
<cluster>
<rm>
	<resources>
    		<service name="test1"/>
    		<service name="test2"/>
		<script name="initscript" file="/etc/init.d/sshd"/>
		<ip address="192.168.1.3" monitor_link="yes"/>
		<ip address="192.168.1.4" monitor_link="yes"/>
    		<fs fstype="ext3" name="mount1" mountpoint="/mnt/cluster" device="/dev/sdb8"/>
    		<fs fstype="ext3" name="mount2" mountpoint="/mnt/cluster2" device="/dev/sdb9"/>
		<nfsexport name="Dummy Export"/>
    		<nfsclient name="User group" target="@users" options="rw,sync"/>
    		<nfsclient name="Admin group" target="@admin" options="rw"/>
    		<nfsclient name="yellow" target="yellow" options="rw,no_root_squash"/>
    		<nfsclient name="magenta" target="magenta" options="rw,no_root_squash"/>
    		<nfsclient name="red" target="red" options="rw"/>
		
	</resources>
	<service ref="test1">
		<ip ref="192.168.1.3"/>
		<script ref="initscript"/>
		<fs ref="mount1">
			<nfsexport ref="Dummy Export">
				<nfsclient ref="Admin group"/>
				<nfsclient ref="User group"/>
				<nfsclient ref="red"/>
			</nfsexport>
		</fs>
	</service>
	<service ref="test2">
		<ip ref="192.168.1.3"/>
		<ip ref="192.168.1.4"/>
		<script ref="initscript"/>
		<fs ref="mount2">
			<nfsexport ref="Dummy Export">
				<nfsclient ref="Admin group"/>
				<nfsclient ref="User group"/>
				<nfsclient ref="red"/>
			</nfsexport>
		</fs>
	</service>
</rm>
</cluster>
