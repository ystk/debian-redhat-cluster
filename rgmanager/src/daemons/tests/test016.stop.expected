Warning: Max references exceeded for resource address (type ip)
Stopping test1...
[stop] script:initscript
[stop] ip:192.168.1.3
[stop] nfsclient:red
[stop] nfsclient:User group
[stop] nfsclient:Admin group
[stop] nfsexport:Dummy Export
[stop] fs:mount1
[stop] service:test1
Stop of test1 complete
Warning: Max references exceeded for resource address (type ip)
Stopping test2...
[stop] script:initscript
[stop] ip:192.168.1.4
[stop] nfsclient:red
[stop] nfsclient:User group
[stop] nfsclient:Admin group
[stop] nfsexport:Dummy Export
[stop] fs:mount2
[stop] service:test2
Stop of test2 complete
